cn_indian_payroll/cn_indian_payroll/overrides/payroll.py

```
@@ -3,16 +3,33 @@
-    #     frappe.msgprint(value)
 
-    # def before_save(self):
-    #     frappe.msgprint(str(self.company))
-    @frappe.whitelist()
-    def payrollset(self,value1):
-        frappe.msgprint(str(value1))
 
+    def on_cancel(self):
+        self.delete_benefit_accrual()
+
+    def delete_benefit_accrual(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `delete_benefit_accrual`

Add return type annotation: `None`

(ANN201)